lib/plugins/aws/deploy/lib/iam-policy-lambda-vpc-template.json

```diff
@@ -0,0 +1,17 @@
+{

```

Comment:
Could you indent this file with 2 spaces (like the other template `.json` files).